pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/JsonConverter.java

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
This file looks very common. Do we want to put them into some kind of `utils` or `common` library?

Reply:
@nlu90 
Yes, this class may have some value and could be moved to utils/common.

But the ES Sink is a separate Pulsar IO Connector.
if we use a class from Pulsar `utils` or `commons` package it means that that class is part of the Public API of Pulsar IO.

I believe that we could move this class to some shared public package in the future, when we can see other users of it.

Does it make sense to you ?

Reply:
Makes